src/lib/plugin_apis/fs.api

```diff
@@ -813,6 +813,20 @@ GType bd_fs_udf_info_get_type () {
  */
 gboolean bd_fs_is_tech_avail (BDFSTech tech, guint64 mode, GError **error);
 
+/**
+ * bd_fs_supported_filesystems:
+ * @error: (out) (optional): currently unused
+ *
+ * Returns: (transfer container) (array zero-terminated=1): list of filesystems supported by this plugin
+ *
+ * Note: This returns filesystems supported by libblockdev, but necessarily
+ *       by the systems this is running on, for this information you need to
+ *       run one if the `bd_fs_can_` functions.
```

Comment:
Parse error. Did you want to say _", but **not** necessarily by the system this is running on"_? and _"run one **of** the `bd_fs_can_` functions."_